dashboard/modules/metrics/metrics_head.py

```diff
@@ -165,7 +165,7 @@ async def prometheus_health(self, req):
                 text = await resp.text()
                 # Basic sanity check of prometheus health check schema
                 # Different flavors of Prometheus may use different health check strings
-                if "Prometheus" not in text or "OK" not in text:
+                if "Prometheus" not in text:
```

Comment:
Hey @richardliaw, just curious, why was the `"OK"` clause removed in this PR? I'm looking into allowing a VictoriaMetrics backend (see [issue](https://github.com/ray-project/ray/issues/42619)) that would ideally also allow a different string to pass the check.

Maybe even more generally, is the `"Prometheus"` check in the response message really needed? It seems very vulnerable to changes in prometheus, and blocks other flavors/implementations of the Prometheus API (e.g. Thanos & VictoriaMetrics)

Reply:
Good point. We'll remove this check in the future and only rely on a 200 response.

Reply:
https://github.com/ray-project/ray/pull/44429